tests/containerprofilecache/helpers_test.go

```
@@ -0,0 +1,148 @@
+	cp *v1beta1.ContainerProfile
+	ap *v1beta1.ApplicationProfile
+	nn *v1beta1.NetworkNeighborhood
+}
+
+var _ storage.ProfileClient = (*stubStorage)(nil)
+
+func newFakeStorage(cp *v1beta1.ContainerProfile) *stubStorage {
+	return &stubStorage{cp: cp}
+}
+
+func (s *stubStorage) setCP(cp *v1beta1.ContainerProfile) {
```

golangci-lint (2.11.4)

[error] func (*stubStorage).setCP is unused

(unused)